Common/cpp/ReanimatedRuntime/WorkletRuntime.h

```diff
@@ -41,7 +41,7 @@ class WorkletRuntime : public jsi::HostObject {
   }
 
   std::string toString() const {
-    return "<WorkletRuntime \"" + name_ + "\">";
+    return "[WorkletRuntime \"" + name_ + "\"]";
```

Comment:
Have you thought about `[ReanimatedWorkletRuntime` ?

Reply:
yes, let's leave it as it is now to avoid changes in the future 😉